Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'annotate', '~> 3.0'
 # Use Sidekiq to enqueue job in the background
 gem 'sidekiq', '~> 6.0.2'
 
+gem 'simplecov', require: false, group: :test
```

Comment:
I'd rather add it within this block https://github.com/rootstrap/rs-code-review-metrics/pull/23/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L56